cibuildwheel/macos.py

```diff
@@ -63,7 +68,7 @@ def call(args, env=None, cwd=None, shell=False):
         call([python, '--version'], env=env)
 
         # install pip & wheel
-        call([python, '-m', 'ensurepip', '--upgrade'], env=env)
+        call([python, get_pip_script, '--no-setuptools', '--no-wheel'], env=env)
```

Comment:
any particular reason for `--no-setuptools` and `--no-wheel`?

Reply:
I'd rather get the script only install pip and let the remaining as it was before.
wheel is installed shortly after.
This is also what travis-ci/dpl is doing for deployment so I guess I just did the same.

Reply:
👍